common/src/lib.rs

```diff
@@ -1,12 +1,13 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 
 cfg_if::cfg_if!{ if #[cfg(feature = "std")] {
-    pub mod emulation_menu;
+    pub mod magenboy_menu;
```

Comment:
Consider merging it with window and moving the conditional compilation into the merged module